mesonbuild/modules/qt4.py

```diff
@@ -127,30 +127,30 @@ def preprocess(self, state, args, kwargs):
             else:
                 basename = os.path.split(rcc_files[0])[1]
                 name = 'qt4-' + basename.replace('.', '_')
-            rcc_kwargs = {'input' : rcc_files,
-                    'output' : name + '.cpp',
-                    'command' : [self.rcc, '-o', '@OUTPUT@', '@INPUT@'],
-                    'depend_files' : qrc_deps,
+            rcc_kwargs = {'input': rcc_files,
+                    'output': name + '.cpp',
+                    'command': [self.rcc, '-o', '@OUTPUT@', '@INPUT@'],
+                    'depend_files': qrc_deps,
```

Comment:
__[flake8]__
* [E128] continuation line under-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->